x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/CsvTestsDataLoader.java

```diff
@@ -315,7 +315,7 @@ private static boolean isLookupDataset(TestDataset dataset) throws IOException {
         return (mode != null && mode.equalsIgnoreCase("lookup"));
     }
 
-    private static boolean isSourceMappingDataset(TestDataset dataset) throws IOException {
```

Comment:
Leftover?

Reply:
Yes, started using it in the query generator, but then I simplified the code.
Reverting now